doc/conf.py

```diff
@@ -15,6 +15,9 @@
 from __future__ import print_function
 import sys
 import os
+
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
@amueller, how can I do this?

I just ran `make doc` and it failed with an error of `six` module not having `PY2` property. I inserted debug output (`__version__` and `__file__`) into `gen_rst.py`, and it turned out, my system's sklearn was used. Can you try the same?